ToyCalendar

ricky-o-k opened the pull request:
1. Base Architecture 정의
- Activity
- Fragment
- Components
  - ViewPager
  - RecyclerView
  - SingleLiveEvent
  - ViewModel
  - Dialog

2. 기본 resource 정의
- color
- dimens
- style (NoActionBar)
- drawable (Dialog)

3. 토의 필요
  TODO 토의 내용1. onAttach / onDetach 작업을 Base에서 진행해주는 것이 맞는가?
  TODO 토의 내용2. 클릭 중복 방지를 이렇게 처리해주는 게 옳은 걸까?

ricky-o-k committed:
ToyCalendar

1. Base Architecture 정의
- Activity
- Fragment
- Components
  - ViewPager
  - RecyclerView
  - SingleLiveEvent
  - ViewModel
  - Dialog

2. 기본 resource 정의
- color
- dimens
- style (NoActionBar)
- drawable (Dialog)

3. 토의 필요
  TODO 토의 내용1. onAttach / onDetach 작업을 Base에서 진행해주는 것이 맞는가?
  TODO 토의 내용2. 클릭 중복 방지를 이렇게 처리해주는 게 옳은 걸까?

ricky-o-k committed:
ToyCalendar

1. Base Architecture 정의
- Activity (* dialog 초기화 코드 빠져서 추가)
- Fragment
- Components
  - ViewPager
  - RecyclerView
  - SingleLiveEvent
  - ViewModel
  - Dialog

2. 기본 resource 정의
- color
- dimens
- style (NoActionBar)
- drawable (Dialog)

3. 토의 필요
  TODO 토의 내용1. onAttach / onDetach 작업을 Base에서 진행해주는 것이 맞는가?
  TODO 토의 내용2. 클릭 중복 방지를 이렇게 처리해주는 게 옳은 걸까?

sk392 commented:
1.  Base Architecture 정의
 -> 일단 지금 상황에서 쓸거말고는 빼는게 어떨까..? ViewPager, Dialog는 현재 스펙에선 굳이 필요 없을 것같아!
 -> Viewmodel에서 AndroidViewmodel로 확장하지 않고 그냥 Viewmodel로 확장하는게 어떨까?? 라이프사이클과 관련이 없는 곳에서도 ViewModel을 쓸 수 있어야될 것같아!

3. 토의내용
3-1. TODO 토의 내용1. onAttach / onDetach 작업을 Base에서 진행해주는 것이 맞는가?
 -> 굳이 진행하지 않아도 될 것같아..!
3-2. TODO 토의 내용2. 클릭 중복 방지를 이렇게 처리해주는 게 옳은 걸까?
 -> 클릭방지 처리가 모든 클릭에서 방지해줘야 하는건 아니니까 필요하다면 해당 클릭에서만 따로 하는게 좋을 것같아 필요하다면 OneClickButton이라는 View를 하나 만드는 것도 괜찮을 것같아!

ricky-o-k commented:
1. Base Architecture 정의
- 여과없이 가져온 컴포넌트도 있긴 한데 다이얼로그는 쓰지 않을까하는 생각이..? 중간중간 제플린 디자인 가이드라인 보니까 보였어서..!! 뷰페이져는 굳이 없어도 되긴 할 것 같음

2. 토의내용
- 만나서 코드를 한번 수정하는 것도 괜찮은 방법이려나?

ricky-o-k committed:
ToyCalendar

1. Base Architecture 정의
- Activity (* dialog 초기화 코드 빠져서 추가)
- Fragment
- Components
  - ViewPager
  - RecyclerView
  - SingleLiveEvent
  - ViewModel
  - Dialog

2. 기본 resource 정의
- color
- dimens
- style (NoActionBar)
- drawable (Dialog)

3. 토의 필요
  TODO 토의 내용1. onAttach / onDetach 작업을 Base에서 진행해주는 것이 맞는가?
  TODO 토의 내용2. 클릭 중복 방지를 이렇게 처리해주는 게 옳은 걸까?

4. 패키지명 오류 수정
buv.co.kr -> yapp.co.kr.toycalendar

ricky-o-k committed:
ToyCalendar

5. BaseViewPager 삭제
